src/bls12_381/fq.rs

```diff
@@ -21,6 +28,14 @@ const R2: FqRepr = FqRepr([0xf4df1f341c341746, 0xa76e6a609d104f1, 0x8de5476c4c95
 // INV = -(q^{-1} mod q) mod q
 const INV: u64 = 0x89f3fffcfffcfffd;
 
+// SWENC_CONST0 = sqrt(-3) mod q =
+// 1586958781458431025242759403266842894121773480562120986020912974854563298150952611241517463240701
+pub const SWENC_CONST0: Fq = Fq(FqRepr([0x1dec6c36f3181f22, 0xb4b9bb641054b457, 0x25695a2be9415286, 0x982b6cbf66c749bc, 0x7d58e1ae1feb7873, 0x62c96300937c0b9]));
```

Comment:
It would be good to document why these constants are significant.

Reply:
right, yes - I added a one-line description. For sure it's not exhaustive, but at least it provides a small indication of why it's needed and where in the paper one can find it.